build.gradle

```
@@ -84,3 +95,16 @@ jar {
     }
     dependsOn(copyToLib)
 }
+
+nexusPublishing {
+    packageGroup = 'org.rundeck-plugins'
```

the group must be `org.rundeck.plugins`
```suggestion
    packageGroup = 'org.rundeck.plugins'
```